File: src/main/java/no/nav/personopplysninger/features/endreopplysninger/domain/EndringTelefon.java
package no.nav.personopplysninger.features.endreopplysninger.domain;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import io.swagger.annotations.ApiModel;

@ApiModel(description = "Responsen fra GET /api/v1/endring/{endringId} når endring gjelder telefonnummer.")
@JsonIgnoreProperties(ignoreUnknown = true)
public class EndringTelefon extends Endring {
    private Telefonnummer innmeldtEndring;

    public Telefonnummer getInnmeldtEndring() {
        return innmeldtEndring;
    }
}

File: src/main/kotlin/no/nav/personopplysninger/features/endreopplysningergammel/domain/telefon/EndringTelefon.kt
package no.nav.personopplysninger.features.endreopplysningergammel.domain.telefon

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import no.nav.personopplysninger.features.endreopplysningergammel.domain.Endring

@JsonIgnoreProperties(ignoreUnknown = true)
class EndringTelefon : Endring<EndringTelefon> {
    var innmeldtEndring: Telefonnummer? = null

    constructor()

    constructor(innmeldtEndring: Telefonnummer) {
        this.innmeldtEndring = innmeldtEndring
    }
}
